src/main/java/dinkplugin/notifiers/data/SpeedrunNotificationData.java

```diff
@@ -1,12 +1,33 @@
 package dinkplugin.notifiers.data;
 
 import lombok.EqualsAndHashCode;
+import lombok.NonNull;
```

Comment:
nit: prefer using `@NotNull` here since lombok's version is meant to impose an actual null-check (rather than a simple type hint)

Reply:
Fixed by [`c78b931` (#329)](https://github.com/pajlads/DinkPlugin/pull/329/commits/c78b93171b9cbc673c0915d449a1da90a2ab89ab)